errorNotificationSender.ps1

```diff
@@ -0,0 +1,95 @@
+Param(
+    [string[]] $logs
+)
+
+$MailSenderType = @' 
+namespace WindowsAPILib
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Net;
+    using System.Net.Mail;
+
+    public static class OutlookDotComMail
+    {
+        public static void SendMail(List<string> mailtoUsers,List<string> mailccUsers , string subject, string sender, string senderPassword, List<string> attachments, string pageBody)
+        {
+            SmtpClient client = new SmtpClient("smtp.office365.com");
+            client.Port = 587;
+            client.DeliveryMethod = SmtpDeliveryMethod.Network;
+            client.UseDefaultCredentials = false;
+            NetworkCredential credentials = new NetworkCredential(sender, senderPassword);
+            client.EnableSsl = true;
+            client.Credentials = credentials;
+
+            try
+            {
+                var mail = new MailMessage();
+                mail.From = new MailAddress(sender.Trim());
+                foreach (var mailtoUser in mailtoUsers)
+                {
+                   var userAddress = new MailAddress(mailtoUser.Trim());
+                   mail.To.Add(userAddress);
+                }
+                foreach (var mailccUser in mailccUsers)
+                {
+                   var userAddress = new MailAddress(mailccUser.Trim());
+                   mail.CC.Add(userAddress);
+                }
+                mail.Subject = subject;
+                mail.IsBodyHtml = true;
+                mail.Body = pageBody;
+                foreach(var attachment in attachments)
+                {
+                   Attachment data = new Attachment(attachment);
+                   mail.Attachments.Add(data);
+                }
+                client.Send(mail);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+                throw ex;
+            }
+        }
+    }
+}
+'@
+Add-Type -TypeDefinition $MailSenderType -Language CSharp;
+
+Function GetEmailContent($logs) {
+    $logFile = Get-Content $logs;
+    $errorLog = $logFile -match "\[error\]";
+    $errorContent = [string]::join("<br/>", $errorLog)
+    $jobLink = "https://ci.appveyor.com/project/xscript/azure-maven-plugins-vvy0i/history"
```

Comment:
what does `azure-maven-plugins-vvy0i`  mean?

Reply:
it's the id of correspond appveyor job